modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc

```diff
@@ -1,250 +1,234 @@
-= Salesforce Marketing Cloud Connector
+= Salesforce Marketing Cloud Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-_Select_
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-The Salesforce Marketing Cloud connector lets you connect to the Marketing Cloud API web services (now known as the Marketing Cloud API), which is also known as the Salesforce Marketing Cloud. This connector exposes convenient operations for exploiting the capabilities of Salesforce Marketing Cloud.
+Salesforce Marketing Cloud Connector Version 3.0.0
+
+Release Notes: xref:release-notes::connector/salesforce-mktg-connector-release-notes-mule-4.adoc[Salesforce Marketing Connector Release Notes]
+
+Anypoint Connector for Salesforce Marketing Cloud (Salesforce Marketing Cloud Connector) lets you connect to the Marketing Cloud API, which is also called Salesforce Marketing Cloud. This connector exposes convenient operations for using the capabilities of Salesforce Marketing Cloud.
 
 == Prerequisites
 
-This document assumes that you are familiar with Salesforce Marketing, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and Global Elements.
+This document assumes that you are familiar with Salesforce Marketing Cloud, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and global elements.
 
 You need login credentials to test your connection to your target resource.
 
 For hardware and software requirements and compatibility
-information, see the Connector Release Notes.
+information, see the connector Release Notes.
```

Comment:
This phrase should be on one line, I think.

Reply:
It looks OK in the preview view.

Reply:
It looks fine in the preview.

Reply:
ok